techniques/general/G56.html

```diff
@@ -68,13 +68,12 @@ <h3>Insufficient Audio Contrast between a narrator and background music</h3>
          <ul>
             <li>#5 is true</li>
          </ul>
-      </section>
```

Comment:
It looks like the inner end section tag (corresponding to Expected Results) was removed in this change?

(As far as I can tell, this should not change; there should still be 2 end section tags here, for Expected Results and Tests)

Reply:
confirmed by looking at this in validator. made the fix